_posts/news/2016-01-25-announcing-scalajs-0.6.6.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: Announcing Scala.js 0.6.6
+category: news
+tags: [releases]
+permalink: /news/2016/01/25/announcing-scalajs-0.6.6/
+---
+
+
+We are excited to announce the release of Scala.js 0.6.6!
+
+It has been a long while since the last release, almost 5 months already.
+That was too long, and we apologize.
+In the future, we hope never to let more than 2 months elapse between consecutive releases.
+
+Besides bug fixes, this release brings several major improvements:
+
+* `js.TupleN`, a JS equivalent of Scala tuples
+* Support for JUnit
+* Better support of constructors of `@ScalaJSDefined` classes:
+  they can now have overloads, default parameters and varargs
+* A completely redesigned internal API for the linker, whose main visible impact should be reduced memory usage and improved speed
+* `js.ConstructorTag[C]`, a `ClassTag` equivalent to `js.constructorOf[C]`
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to
+[the tutorial]({{ BASE_PATH }}/tutorial/).
+
+## Release notes
+
+As a minor release, 0.6.6 is (almost, see Breaking changes below) backward source and binary compatible with previous releases in the 0.6.x series.
+Libraries compiled with earlier versions can be used with 0.6.6 without change.
+However, it is not forward compatible: libraries compiled with 0.6.6 cannot be used by projects using 0.6.{0-5}.
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Breaking changes
+
+### Running in Node.js/PhantomJS
+
+Until Scala.js 0.6.5, the following sbt setting would, as a side-effect, enable Node.js or PhantomJS:
+
+    scalaJSStage in Global := FastOptStage
+
+As of Scala.js 0.6.6, this setting is redundant, as it is the default, but `run` and `test` will still use Rhino by default!
+To disable Rhino and use Node.js/PhantomJS in 0.6.6, use the following setting instead:
+
+    scalaJSUseRhino in Global := false
+
+### Tools API
+
+The [Tools API]({{ site.production_url }}/api/scalajs-tools/0.6.6/#org.scalajs.core.tools.package),
+(the API to the Scala.js linker and optimizer) have been completely redesigned.
+Users of this API, such as Scala.jsFiddle, will experience breakages.
```

Comment:
I made it into a link, so at least it doesn't have the "is there a dot/comma/space?" problem